vendor/modules.txt

```diff
@@ -184,6 +184,8 @@ github.com/huin/goupnp/httpu
 github.com/huin/goupnp/scpd
 github.com/huin/goupnp/soap
 github.com/huin/goupnp/ssdp
+# github.com/imdario/mergo v0.3.12
+github.com/imdario/mergo
```

Comment:
is it normal to have it commented and not commented?

Reply:
No idea :thinking: